packages/cli/package.json

```diff
@@ -83,10 +82,10 @@
     "@types/papaparse": "^5.2.3",
     "@types/plurals-cldr": "^1.0.1",
     "mockdate": "^3.0.2",
-    "typescript": "^4.0.3"
+    "typescript": "^4.0.3",
+    "fs-extra": "^9.0.1"
   },
   "peerDependencies": {
-    "@babel/core": "^7.0.0",
     "babel-plugin-macros": "2 || 3",
```

Comment:
this should be moved to dependencies, instead of peer, as for now we are not relaying on the users environment but providing all needed dependencies by ourselves.